beprof/curve.py

```diff
@@ -164,23 +189,31 @@ def main():
     print('X:', k.x)
     print('Y:', k.y)
     print('M:', k.metadata)
-    print(k)
 
-    print('\n', '*'*30,'\nchange_domain:')
+    print('\n__str__:\n', k)
+    print('Futher tests:\n')
+
+    k2 = k.view(np.ndarray)
+    print(k2)
+
+    k3 = k[1:2,:]
+    print(k3)
+
+    print('\nSubtract method :\n')
+
+    c = Curve([[0.0, 0], [5, 5], [10, 0]])
```

Comment:
Also here something temporary:
I forced datatype to be float, if I just put 0 instead of 0.0 the result will be wrong, because 
if datatype of self.x is int, operation in line 165 will convert to int as well, and in some cases should be float instead.